client/js/InitVue.js

```diff
@@ -79,7 +57,45 @@ function initialize (components = {}, storeModules = {}, apiStoreModules = [], p
         }, 5)
       }
     })
-    Promise.resolve(vm)
+
+    app.config.globalProperties.dplan = window.dplan
+    app.config.globalProperties.Routing = window.Routing
+    app.config.globalProperties.Translator = window.Translator
+    app.config.globalProperties.hasPermission = window.hasPermission
+    app.config.globalProperties.h = window.h
+    app.config.productionTip = false
+
+    if (dplan?.settings?.debug) {
+      app.config.performance = false
+    }
+
+    loadSentry()
+
+    app.directive('dp-validate-multiselect', dpValidateMultiselectDirective)
+    // app.directive('tooltip', Tooltip)
+
+    // Add plugins to Vue instance
+    app.use(PortalVue)
+    app.use(DPVueCorePlugin)
+
+    // Register components that are used globally
+    app.component('DpObscure', DpObscure)
+    app.component('HamburgerMenuButton', HamburgerMenuButton)
+    app.component('NotifyContainer', NotifyContainer)
+    app.component('DpAccordion', DpAccordion)
+    app.component('DpFlyout', DpFlyout)
+
+    Object.values(components).forEach(comp => {
+      if (comp) {
+        app.component(comp.name, comp)
+      } else {
+        console.log('at least one is undefined', components)
+      }
```

Comment:
```suggestion
    Object.keys(components).forEach(comp => {
      if (components[comp]) {
        app.component(components[comp].name, components[comp])
      } else {
        console.log(`${components[comp]} is undefined}`, components)
      }
```

Reply:
Just out of curiosity, wouldn't it be beneficial if we could determine the problematic component by adjusting this logic a little bit like suggested?